internal/scraper/scraper.go

```diff
@@ -524,6 +532,18 @@ func (s Scraper) collectData(ctx context.Context, t time.Time) (*probeData, time
 		timeout = time.Duration(s.check.Timeout) * time.Millisecond
 	}
 
+	ctx, checkSpan := s.tracerProvider.Tracer("").Start(ctx, "check")
+	defer checkSpan.End()
+	checkSpan.SetAttributes(
+		attribute.String("type", s.prober.Name()),
+		attribute.String("region", s.probe.Region),
```

Comment:
This seems to be the probe region (as expected), not the tenant region. Not sure where to get the latter from.